canonical-web-and-design/ubuntu.com
# Convert app index pages to use yaml

## Done

- Convert app index pages to use yaml and markdown templates
- I added all the text and images to the index.md files and all the logic to the _base_appliances_index.html

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/appliance/openhab, http://0.0.0.0:8001/appliance/plex or http://0.0.0.0:8001/appliance/adguard
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
